src/main/resources/org/jenkins/plugins/lockableresources/actions/LockableResourcesRootAction/tableQueue/table.jelly

```diff
@@ -23,11 +23,22 @@ THE SOFTWARE.
 -->
 <?jelly escape-by-default='true'?>
 <j:jelly
+  xmlns:f="/lib/form"
   xmlns:j="jelly:core"
   xmlns:l="/lib/layout"
   xmlns:st="jelly:stapler">
   <st:adjunct includes="io.jenkins.plugins.data-tables"/>
 
+  <!-- translations for javascript--> 
+  <template id="i18n"
+            data-queue-title="${%queue.change.title}"
+            data-queue-message="${%queue.change.message}"
+            data-queue-on-success="${%queue.change.on.success}"
+            data-queue-on-fail="${%queue.change.on.fail}"
+            data-action-on-success="${%action.on.success}"
+            data-action-on-fail="${%action.on.fail}"
+             />
+
   <j:set var="queue" value="${it.getQueue()}"/>
 
   <div class="row py-3">
```

Comment:
Probably out-of-scope for this PR, but in case there is no message shown, this `<div class="row py-3">` will introduce a weird vertical white-space.

You could try adding a class to the `div` and adding the CSS:
```
.myclass:empty {
  display: none;
}
```
Should be safe to use: https://caniuse.com/mdn-css_selectors_empty